src/customer/strategies/googlepay/googlepay-customer-initialize-options.ts

```diff
@@ -0,0 +1,21 @@
+export default interface GooglePayCustomerInitializeOptions {
+    /**
+     * This container is used to set an event listener, provide an element ID if you want
+     * users to be able to launch the GooglePay wallet modal by clicking on a button.
+     * It should be an HTML element.
+     */
+    container: string;
+
+    /**
+     * A callback that gets called when GooglePay fails to initialize or
+     * selects a payment option.
+     *
+     * @param error - The error object describing the failure.
+     */
+    onError?(error: Error): void;
+
+    /**
+     * A callback that gets called when the customer selects a payment option.
+     */
+    onPaymentSelect?(): void;
```

Comment:
Do we even honour this option? I don't think it ever gets called. Either honour it or remove it so we don't mislead the user.